### PR TITLE
Put in place triple-equals, restrict-plus-operands; ignore no-unnecessary-local

### PR DESCRIPTION
**Pull request by PrashanthCorp**

### CODE REVIEW

File: src/graph/GraphViewServer.ts [hunk 1]
```diff
@@ -436,6 +436,7 @@ export class GraphViewServer extends EventEmitter {
   private handleQueryMessage(queryId: number, gremlin: string) {
     this.log(`Query requested: queryId=${queryId}, gremlin="${gremlin}"`);
 
+    //tslint:disable-next-line:no-floating-promises
     this.queryAndShowResults(queryId, gremlin);
```
Comment (PrashanthCorp) [inline src/graph/GraphViewServer.ts]: `handleQueryMessage` is not async, and is used in the GraphViewServerSocket. 
@StephenWeatherford should this be awaited?

File: tslint.json [hunk 2]
```diff
@@ -162,7 +162,7 @@
         "no-suspicious-comment": true,
         "no-typeof-undefined": true,
         "no-unnecessary-field-initialization": true,
-        // TODO: "no-unnecessary-local-variable": true,
+        "no-unnecessary-local-variable": false, //changed
```
Comment (PrashanthCorp) [inline tslint.json]: Not enforcing this, as per discussion with @StephenWeatherford

File: src/graph/GraphViewServer.ts (hunk 1 above)
Comment (StephenWeatherford) [inline src/graph/GraphViewServer.ts]: Why are you thinking this one should be left alone?

Comment (StephenWeatherford) [reply src/graph/GraphViewServer.ts]: Could you add return value to handleQueryMessage?

Makes sense, can't handle this synchronously.

FYI, that would normally be a red flag - it means we could get overlapping calls.  Luckily this is already being handled at least somewhat - I associate each query with an ID, and the client ignores results that are not from the most recent query, so we're probably fine here.

**Comment by StephenWeatherford** [comment]
You gonna merge this?

**Comment by PrashanthCorp** [comment]
Merging. I expected some merge conflicts & build failures for the other PR's when this is merged, but this has probably waited enough

**Pull request merged**